Description: Fix the manual page errors - replace mdoc macros with man ones.
Forwarded: no
Author: Peter Pentchev <roam@ringlet.net>
Last-Update: 2010-07-02

--- a/doc/sockd.conf.5
+++ b/doc/sockd.conf.5
@@ -420,10 +420,10 @@
 The syntax of the redirect statement is
 as follows:
 
-.D1
+.IP
 \fBredirect\fP from: \fBADDRESS\fP
 
-.D1
+.IP
 \fBredirect\fP to: \fBADDRESS\fP
 
 The semantics of \fBfrom\fP and \fBto\fP vary according to
@@ -466,8 +466,8 @@
    Karl-Andre' Skevik
 .SH SEE ALSO
 danted(8), dante.conf(5), hosts_access(5)
-.Pp
+.PP
 Information about new releases and other related issues can be found
 on the
-.Nm \fBDante\fP
+.B \fBDante\fP
 WWW home page: http://www.inet.no/dante/
--- a/doc/socksify.1
+++ b/doc/socksify.1
@@ -123,7 +123,7 @@
    Karl-Andre' Skevik
 .SH SEE ALSO
 socks.conf(5) sockd(8) sockd.conf(5)
-.Pp
+.PP
 Information about new releases and other related issues can be found
 on the
 \fBDante\fP
